pages/game/roleScreen.tsx

```diff
@@ -8,16 +8,28 @@ interface RoleProps {
 }
 
 const RoleScreen: React.FC<RoleProps> = ({ data }) => {
-  const userId = useUserId();
 
   if (!data) {
     return (<></>);
   }
 
+  const unveilRole = ()=>{
+    const roleCard = document.querySelector(".roleCard")
+    console.log("REVEALED", roleCard, data)
```

Comment:
We don't want to put console logs into 'prod'

Reply:
I was gonna use it to make sure everything was actually working, lol.

Removed